app/javascript/components/tooltips/Concept.tsx

```diff
@@ -0,0 +1,244 @@
+  parentSlug?: string
+}): JSX.Element => {
+  const tooltipId = `concept-tooltip${
+    parentSlug ? `-${parentSlug}` : parentSlug
+  }`
+  // Retrieve the HTML contents from the endpoint
+  const { isLoading, isError, data: html_content } = useQuery(tooltipId, () =>
+    fetch(endpoint).then((res) => res.text())
+  ) as { isLoading: boolean; isError: boolean; data: string }
+
+  const tooltipRef = useRef(null)
+  const [{ show }, showDispatcher] = useReducer(showReducer, {
+    requestHover: false,
+    requestFocus: false,
+    hover: false,
+    focus: false,
+    show: false,
+  })
+
+  const popper = usePopper(parent, tooltipRef.current, {
+    placement: 'bottom',
+    modifiers: [{ name: 'offset', options: { offset: [0, 8] } }],
+  })
+
+  // For the next two useEffects:
+  // If I ignore the exhaustive deps warning and only have requestToShow, it
+  // works.  If I have the exhaustive deps added (popper) it causes inf loop
+  // due to popper reference changing on every render.
+
+  useEffect(() => {
+    if (hoverRequestToShow && popper?.update) {
+      popper.update()
+      dispatch(tooltipId, showDispatcher, 'request-hover', 'show')
+      return
+    }
+
+    setTimeout(
+      () => dispatch(tooltipId, showDispatcher, 'request-hover', 'hide'),
+      150
+    )
+  }, [hoverRequestToShow]) // eslint-disable-line react-hooks/exhaustive-deps
+
+  useEffect(() => {
+    if (focusRequestToShow && popper?.update) {
+      popper.update()
+      dispatch(tooltipId, showDispatcher, 'request-focus', 'show')
+      return
+    }
+
+    dispatch(tooltipId, showDispatcher, 'request-focus', 'hide')
+  }, [focusRequestToShow]) // eslint-disable-line react-hooks/exhaustive-deps
+
+  // Get styles and classes to apply
+  const styles = { ...popper.styles.popper, zIndex: 10 }
+  const classNames = ['c-tooltip', 'c-concept-tooltip']
+  if (!show) {
+    classNames.push('tw-invisible')
+  }
+  if (show) {
+    classNames.push('tw-pointer-events-auto')
+  }
+
+  // Separate render exists because happy path needs to render
+  // 'dangerouslySetInnerHTML' and this does not.
+  if (isLoading || isError) {
+    return (
+      <div
+        ref={tooltipRef}
+        className={classNames.join(' ')}
+        style={styles}
+        {...popper.attributes.popper}
+        role="tooltip"
+      >
+        {isLoading && <Loading />}
+        {isError && <p>Something went wrong</p>}
+      </div>
+    )
+  }
+
+  return (
+    <div
+      ref={tooltipRef}
+      className={classNames.join(' ')}
+      style={styles}
+      {...popper.attributes.popper}
+      role="tooltip"
+      tabIndex={show ? undefined : -1}
+      onFocus={() => {
+        dispatch(tooltipId, showDispatcher, 'focus', 'show')
+      }}
+      onBlur={() => {
+        dispatch(tooltipId, showDispatcher, 'focus', 'hide')
+      }}
+      onMouseEnter={() => dispatch(tooltipId, showDispatcher, 'hover', 'show')}
+      onMouseLeave={() => dispatch(tooltipId, showDispatcher, 'hover', 'hide')}
```

Comment:
As far as I can tell, `dispatch` is unneccessary and can be replaced as follows

```suggestion
      onFocus={() => { showDispatcher(tooltipId, 'focus', 'show') }}
      onBlur={() => { showDispatcher(tooltipId, 'focus', 'hide') }}
      onMouseEnter={() => showDispatcher(tooltipId, 'hover', 'show')}
      onMouseLeave={() => showDispatcher(tooltipId, 'hover', 'hide')}
```